JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/videoPlayer/VideoPlayerScreen.kt

```diff
@@ -160,12 +164,13 @@ fun VideoPlayerScreenContent(movieDetails: MovieDetails, onBackPressed: () -> Un
             )
             .focusable()
     ) {
-        AndroidView(
-            factory = {
-                PlayerView(context).apply { useController = false }
-            },
-            update = { it.player = exoPlayer },
-            onRelease = { exoPlayer.release() }
+        PlayerSurface(
+            exoPlayer,
+            SURFACE_TYPE_SURFACE_VIEW,
```

Comment:
I'd recommend using named parameters (because in the future we're actually changing this API to use a default version for surface type, which means the parameter order will change, effectively breaking your current implementation)

Reply:
Thanks for the suggestion. Changed the code to use named parameters.